selfdrive/manager/process_config.py

```diff
@@ -8,7 +8,7 @@
 WEBCAM = os.getenv("USE_WEBCAM") is not None
 
 def driverview(started: bool, params: Params, CP: car.CarParams) -> bool:
-  return params.get_bool("IsDriverViewEnabled")  # type: ignore
+  return started or params.get_bool("IsDriverViewEnabled")
 
 def notcar(started: bool, params: Params, CP: car.CarParams) -> bool:
   return CP.notCar  # type: ignore
```

Comment:
```suggestion
  return started and CP.notCar  # type: ignore
```

Reply:
outdated before you could even comment :)